ReactSkia/sdk/CurlNetworking.cpp

```diff
@@ -326,7 +332,9 @@ bool CurlNetworking::sendRequest(shared_ptr<CurlRequest> curlRequest, folly::dyn
   return status;
 } 
 bool CurlNetworking::abortRequest(shared_ptr<CurlRequest> curlRequest) {
+  std::scoped_lock lock(mutex_);
```

Comment:
whether this lock to be done, before removing handle or irrespective of the handle, when abortRequest is posted